runtime/near-vm-runner/src/prepare.rs

```diff
@@ -491,7 +481,7 @@ mod tests {
     fn imports() {
         // nothing can be imported from non-"env" module for now.
         let r =
-            parse_and_prepare_wat(r#"(module (import "another_module" "memory" (memory 1 1)))"#);
```

Comment:
Not great that this test needed to be adapted. Can we move the relevant check to before we modify the module?